sci-mathematics/yafu/files/yafu-1.34.3-seds.patch

```diff
@@ -0,0 +1,57 @@
+Weres seds from ebuild:
+	sed -i \
+		-e 's:../gmp/include:gmp:' \
+		-e 's:../gmp-ecm/include:gmp-ecm:' \
+		-e 's:# LIBS += -L../msieve/lib/linux/x86_64:LIBS += -lmsieve -lz -ldl:' \
+		-e 's:CFLAGS = -g:#CFLAGS = -g:' \
+		-e '/$(LIBS)$/s:$(CC):$(CC) $(LDFLAGS):g' Makefile || die
+	sed -i -e 's:\"config.h\":<gmp-ecm/config.h>:g' top/driver.c || die
+
+	# proper ggnfs default path
+	sed -i -e 's~strcpy(fobj->nfs_obj.ggnfs_dir,"./");~strcpy(fobj->nfs_obj.ggnfs_dir,"/usr/bin/");~' factor/factor_common.c || die
+--- a/Makefile
++++ b/Makefile
+@@ -20,16 +20,16 @@
+ #CC ?= gcc
+ #CC = x86_64-w64-mingw32-gcc-4.5.1
+ #CFLAGS = -march=core2 -mtune=core2
+-CFLAGS = -g
++#CFLAGS = -g
+ WARN_FLAGS = -Wall # -Wconversion
+ #OPT_FLAGS = -O3
+ INC = -I. -Iinclude
+ 
+ # modify these for your particular gmp/gmp-ecm installation
+-INC += -I../gmp/include
++INC += -Igmp
```

Comment:
Is this bundling gmp and gmp-ecm?

Reply:
As far as I can see, no. I think those are debris from building this program on windows.